flink-filesystems/flink-s3-fs-presto/src/main/resources/META-INF/NOTICE

```diff
@@ -6,32 +6,38 @@ The Apache Software Foundation (http://www.apache.org/).
 
 This project bundles the following dependencies under the Apache Software License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt)
 
+- aopalliance:aopalliance:1.0
 - commons-beanutils:commons-beanutils:1.9.4
 - commons-codec:commons-codec:1.13
 - commons-collections:commons-collections:3.2.2
 - commons-io:commons-io:2.8.0
 - commons-lang:commons-lang:2.6
 - commons-logging:commons-logging:1.1.3
-- com.amazonaws:aws-java-sdk-core:1.11.788
-- com.amazonaws:aws-java-sdk-dynamodb:1.11.788
-- com.amazonaws:aws-java-sdk-kms:1.11.788
-- com.amazonaws:aws-java-sdk-s3:1.11.788
-- com.amazonaws:aws-java-sdk-sts:1.11.788
-- com.amazonaws:jmespath-java:1.11.788
-- com.facebook.presto:presto-hive:0.187
+- com.amazonaws:aws-java-sdk-core:1.11.951
+- com.amazonaws:aws-java-sdk-dynamodb:1.11.951
+- com.amazonaws:aws-java-sdk-kms:1.11.951
+- com.amazonaws:aws-java-sdk-s3:1.11.951
+- com.amazonaws:aws-java-sdk-sts:1.11.951
+- com.amazonaws:jmespath-java:1.11.951
+- com.facebook.presto:presto-common:0.257
+- com.facebook.presto:presto-hive:0.257
+- com.facebook.presto:presto-hive-common:0.257
+- com.facebook.presto:presto-hive-metastore:0.257
 - com.facebook.presto.hadoop:hadoop-apache2:2.7.3-1
 - com.fasterxml.jackson.core:jackson-annotations:2.12.1
 - com.fasterxml.jackson.core:jackson-core:2.12.1
 - com.fasterxml.jackson.core:jackson-databind:2.12.1
 - com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:2.12.1
 - com.fasterxml.woodstox:woodstox-core:5.0.3
-- com.google.guava:guava:21.0
-- io.airlift:configuration:0.153
-- io.airlift:log:0.153
-- io.airlift:stats:0.153
-- io.airlift:units:1.0
-- io.airlift:slice:0.31
+- com.google.guava:guava:26.0-jre
+- com.google.inject:guice:4.2.2
```

Comment:
guice was excluded previously, but is now directly referenced in presto-hive. Although it seemed to have worked anyway, I removed the exclusion on it now.